content/MenuButton.qml

```diff
@@ -10,27 +14,31 @@ MainScreeen {
         anchors.topMargin: 100
         text: "3DCar"
     }
+
     Button {
         id: acButton
         anchors.top: parent.top
         anchors.horizontalCenter: parent.horizontalCenter
         anchors.topMargin: 175
```

Comment:
Use only `anchors.horizontalCenter: parent.horizontalCenter` and `topPadding`, `spacing` properties for column